### PR TITLE
Adding LEDs, HSM and watchdog tags to spec generation

### PR DESCRIPTION
**Pull request by rvs**
Tiny tweak to the spec generation script based on EVE TSC discussion

### CODE REVIEW

File: pkg/debug/spec.sh [hunk 1]
```diff
@@ -72,6 +72,8 @@ else
 fi
 
 DISK=$(lsblk -b  | grep disk | awk '{ total += $4; } END { print int(total/(1024*1024*1024)); }')
+WDT=$([ -e /dev/watchdog ] && echo true || echo false)
+HSM=$([ -e /dev/tpm0 ] && echo 1 || echo 0)
```
Comment (eriknordmark) [inline pkg/debug/spec.sh:76]: /dev/tpm0 is TPM 1.2.
Need to look for /dev/tpmrm0 for TPM 2.0

File: pkg/debug/spec.sh [hunk 2]
```diff
@@ -81,7 +83,10 @@ cat <<__EOT__
   "attr": {
     "memory": "${MEM}G",
     "storage": "${DISK}G",
-    "Cpus": "${CPUS}"
+    "Cpus": "${CPUS}",
+    "watchdog": ${WDT},
+    "hsm": ${HSM},
+    "leds": 0
```
Comment (eriknordmark) [inline pkg/debug/spec.sh:89]: Was the intent to look in /sys/class/leds/ just to return the count of leds?

**Review by eriknordmark — CHANGES_REQUESTED** [review]
Need to fix the /dev/tpm reference

File: pkg/debug/spec.sh (hunk 1 above)
Comment (rvs) [reply pkg/debug/spec.sh:76]: Fixed

File: pkg/debug/spec.sh (hunk 2 above)
Comment (rvs) [reply pkg/debug/spec.sh:89]: The problem with looking there is that most of the time you'll get confused by the LEDs that are present on the keyboard (sine they get registered there) and by things like mmcX leds (which are not really visible). Hence for now this is just a placeholder for a human being to edit it.

**Comment by rvs** [comment]
Hey @eriknordmark -- can you please take a quick look?

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**